scripts/scripts/testing/automations/incremental/nigeria.py

```diff
@@ -0,0 +1,38 @@
+import pandas as pd
+import requests
+from bs4 import BeautifulSoup
+from datetime import date
+
+
+def main():
+    data = pd.read_csv("automated_sheets/Nigeria.csv").sort_values(by="Date", ascending=False)
```

Comment:
To ensure the path works regardless of where the script is executed, I'd suggest using function `get_project_dir` as:

```python
path = os.path.join(get_project_dir(), "scripts", "scripts", "testing", "automated_sheets", "Nigeria.csv")
data = pd.read_csv(path).sort_values(by="Date", ascending=False)
```

Note you need to import:

```python
import os
from cowidev.utils.utils import get_project_dir
```